src/Umbraco.Web.UI.Client/src/less/installer.less

```diff
@@ -321,6 +321,9 @@ select {
 
   .noUi-value {
     cursor: pointer;
+    &-active{
+      color: @black;
+    }
```

Comment:
I think we could include `umb-range-slider` less here:
https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/less/installer.less

and then add the class `umb-range-slider` to the slider in installer user step.
Then we can probably remove all this redundant styling: https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/less/installer.less#L288-L329

It styling of the range slider later change, we ensure the styling is consistent.

Reply:
I agree with @bjarnef. Would be very nice if that is possible.

Reply:
I think this should work by adding this to intaller.less:

```
@import "components/umb-range-slider.less";
```

and remove the noUiSlider specific styling  here:
https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/less/installer.less#L288-L329

and finally this update:

```
<div id="consentSlider" class="umb-range-slider"></div>
```

Reply:
I've moved the styling into umb-range-slider.less